charts/cbcontainers-operator/README.md

```diff
@@ -39,7 +25,7 @@ helm repo update
 helm install cbcontainers-operator TODO-chart-repo/TODO-chart-name --namespace X
```

Comment:
If we have these values; can we fix that now too ?

Reply:
yes and no. I think the actual repo we push the chart to is private (need to check for sure). so it's better to leave these as they are (or just say to install from source) rather than link to a private repo. that is, until we can find a public repo for the charts.